_translator-files/po/sv-SE/Include-Client-Commands.po

```diff
@@ -1,52 +1,52 @@
 # SOME DESCRIPTIVE TITLE
 # Copyright (C) YEAR Free Software Foundation, Inc.
 # This file is distributed under the same license as the PACKAGE package.
-# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
-#
-#, fuzzy
+# Daniel <daniel@mailsystem.net>, 2024.
 msgid ""
 msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"PO-Revision-Date: 2024-08-05 13:09+0000\n"
+"Last-Translator: Daniel <daniel@mailsystem.net>\n"
 "Language-Team: LANGUAGE <LL@li.org>\n"
-"Language: \n"
+"Language: sv-SE\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=n != 1;\n"
+"X-Generator: Weblate 5.7-dev\n"
 
 #. type: Bullet: '- '
 #: ../wiki/en/Include-Client-Commands.md:3
 #, markdown-text
 msgid "`-M` or `--mutestream` Prevent others on a server from hearing what I play"
-msgstr ""
+msgstr "`-M` eller `--mutestream` Förhindra att andra på servern hör vad jag spelar"
 
 #. type: Bullet: '- '
 #: ../wiki/en/Include-Client-Commands.md:3
 #, markdown-text
 msgid "`--mutemyown` Prevent me from hearing what I play in the server mix (headless only)"
-msgstr ""
+msgstr "`--mutemyown` Förhindra att jag hör vad jag spelar i serverblandningen (headless endast)"
 
 #. type: Bullet: '-  '
 #: ../wiki/en/Include-Client-Commands.md:3
 #, markdown-text
 msgid "`-c` or `--connect` Connect to given server address on startup, format `address[:port]`"
-msgstr ""
+msgstr "`-c` eller `--connect` Anslut till angiven serveradress vid uppstart, format `address[:port]`"
 
 #. type: Bullet: '-  '
 #: ../wiki/en/Include-Client-Commands.md:4
 #, markdown-text
 msgid "`-j` or `--nojackconnect` Disable auto JACK connections"
-msgstr ""
+msgstr "`-j` eller `--nojackconnect` Inaktivera automatisk JACK-anslutning"
 
 #. type: Bullet: '-  '
 #: ../wiki/en/Include-Client-Commands.md:6
 #, markdown-text
 msgid "`--ctrlmidich` MIDI controller channel to listen on, control number offset and consecutive CC numbers (channels) and Mute Myself CC number. Format: `channel[;f*][;p*][;s*][;m*][;o]` See [Tips & Tricks](Tips-Tricks-More#using-ctrlmidich-for-midi-controllers)"
-msgstr ""
+msgstr "`--ctrlmidich` MIDI-kontrollerkanal att lyssna på, kontrollnummer-offset och efterföljande CC-nummer (kanaler) och Mute Myself CC-nummer. Format: `channel[;f*][;p*][;s*][;m*][;o]` Se [Råd & dåd](Tips-Tricks-More#using-ctrlmidich-for-midi-controllers)"
```

Comment:
Translate text after # in link.

Reply:
`använder-ctrlmidich-för-midi-kontroller`